miner.go

```diff
@@ -128,6 +128,12 @@ func newSoloMiner(ctx context.Context, devices []*Device) (*Miner, error) {
 	return m, nil
 }
 
+func newBenchmarkMiner(devices []*Device) (*Miner, error) {
```

Comment:
Should not return an error here since there is no possible error as pointed out by the `unparam` CI failure.

Reply:
Updated.